src/kohlrahbi/__init__.py

```diff
@@ -231,6 +242,39 @@ def main(pruefis: list[str], input_path: Path, output_path: Path, file_type: lis
             del unfolded_ahb
         gc.collect()
 
+    if "conditions" in file_type:
+        # store conditions in conditions.json files
+        dump_conditions_json(output_directory_path=output_path, already_known_conditions=collected_conditions)
+
+
+def dump_conditions_json(output_directory_path: Path, already_known_conditions: dict) -> None:
+    """
+    Writes all collected conditions to a json file.
+    The file will be stored in the directory:
+        'output_directory_path/<edifact_format>/conditions.json'
+    """
+    for edifact_format in already_known_conditions:
+        condition_json_output_directory_path = output_directory_path / str(edifact_format)
+        condition_json_output_directory_path.mkdir(parents=True, exist_ok=True)
+        file_path = condition_json_output_directory_path / "conditions.json"
+
+        sorted_condition_dict = {
+            k: already_known_conditions[edifact_format][k]
+            for k in sorted(already_known_conditions[edifact_format], key=int)
+        }
+        array = [
+            {"condition_key": i, "condition_text": sorted_condition_dict[i], "edifact_format": edifact_format}
```

Comment:
Du kannst hier auch die Typisierung aus ahbicht verwenden, falls (und nur Falls) wir ahbicht eh schon als dependency haben (falls nicht, dann lohnt es nicht dafür).
schreib in letzterem Fall aber zumindest dazu, dass es sich um ConditionKeyConditionTextMappings handelt.

Reply:
e287ec65fc9dbfd9ad45dddf208b18301f9e61fc